workflows/iuf/operations/s3-upload/s3-upload-template.yaml

```
@@ -136,3 +156,23 @@ spec:
       image: artifactory.algol60.net/csm-docker/stable/docker.io/alpine/git:2.32.0
       command: [sh, -c]
       args: ["DIFF_TIME=$(expr {{inputs.parameters.opend}} - {{inputs.parameters.opstart}}); echo $DIFF_TIME; echo $DIFF_TIME > /tmp/diff_time.txt"]
+  
+  - name: make-product-catalog-content
+    inputs:
+      parameters:
+      - name: manifest_s3_content
+    outputs:
+      parameters:
+      - name: product-content
+        valueFrom:
+          path: /tmp/product-content
```

I'm actually going to leave one more suggestion. As an extra layer of protection to ensure that we don't fail when the script unexpectedly fails to write out /tmp/product-content, we can add a default value here.

```suggestion
        valueFrom:
          path: /tmp/product-content
          default: "{}"
```

---

I fixed this, but not the way Eli suggested.